docs/sources/reference/api/docker_remote_api.md

```diff
@@ -46,6 +46,22 @@ You can still call an old version of the API using
 
 ### What's new
 
+`GET /containers/(id)/copy`
+
+`PUT /containers/(id)/copy`
```

Comment:
It's really weird though... because the existing `POST` endpoint should have been a `GET`. The new "copy files/folders to a container" endpoint is a `PUT` because, as you mentioned above, is the ReSTful thing to do. I think any user of an HTTP API would think that it's still simple, and HTTP ReST purists everywhere will be happier using `PUT`.

If we *did* use `POST` instead of `PUT` for this endpoint then the behavior will be the opposite of the original since the original returned contents from the container and now it would only be used to send contents to the container. If you guys are okay with this then I'll update it. :)

and make the `copy-accross` endpoint be a `POST` also.